### PR TITLE
Makes it possible to pass options to htmlparser2

### PR DESCRIPTION
**Pull request by matsrorbecker**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const Browser = require("../");
 const nock = require("nock");
 const Path = require("path");
 const Script = require("@bonniernews/wichita");
+const { expect } = require("chai");
```
Comment (jonaswalden) [inline test/index.js:8]: boo!!

**Review by jonaswalden — CHANGES_REQUESTED** [review]

File: test/index.js [hunk 2]
```diff
@@ -425,4 +425,19 @@ describe("Tallahassee", () => {
...
+  describe("options", () => {
+    describe("htmlparser2 options", () => {
+      it("passes on options to htmlparser2", async () => {
+        const markup = "<svg><text>åäö</text></svg>";
+        const htmlparser2 = {
+          xmlMode: true,
+          decodeEntities: true
+        };
+        const browser = await new Browser({ htmlparser2 }).load(markup);
+        expect(browser.$.html()).to.not.include("<html>");
```
Comment (andolf) [inline test/index.js:438]: What is the purpose of this? Tallahassee should mimic a browser, so I would expect it to wrap its content in html.

Comment (jonaswalden) [reply test/index.js:438]: the assertion is just to make sure that options are passed along.
the actual purpose of enabling this is to verify that no illegal (ESI) dollar signs are output in solveig. with `encodeEntities: false` we can't see a difference between `&dollar;` and an actual `$`. it's actually the only option we need but figured it'd be less complex just passing along entire options object.

**Review by jonaswalden — COMMENTED** [review]
yay!

**Comment by jonaswalden** [comment]
😞 never mind.
this wasn't the issue.

**Pull request closed without merging**